utils/libgds.py

```diff
@@ -0,0 +1,122 @@
+# Standard imports
+import logging
+import requests
+from tempfile import NamedTemporaryFile
+from datetime import datetime
+
+# ICA imports
+from libica.openapi import libgds
+from libica.openapi.libgds.rest import ApiException
+
+# Utils
+from utils import ica
+
+# Get logger
+logger = logging.getLogger()
+
+
+def get_gds_file_id_from_gds_path(volume_name: str, path: str) -> (str, None):
+    """
+    From a gds path, collect the file id.
+    One can then use the file id to collect other metadata on a file
+    """
+
+    configuration = libgds.Configuration(
+        host="https://aps2.platform.illumina.com",
+        api_key={
+            'Authorization': 'YOUR_API_KEY'
+        }
+    )
```

Comment:
This is not used? Meant as comment?

Reply:
Resolved in 495ec2001399f8b6125a8b4b6e3cd544794d3d82